### ISSUE TITLE
CanBeNull / NotNull annotations support

### ISSUE THREAD

**Issue by t-denis** [SergeyTeplyakov/ReSharperContractExtensions#54]
Hi!
What do you think about an option to automatically add CanBeNull/NotNull annotations when they are available?
Like R# does with it`s default quick-fix "Check parameter for Null".

**Comment by tom-englert** [comment]
Good point - I was already thinking about to suggest the same feature. 

Currently I have turned off R#'s notifications about null checks, but it would be great to have both CC and R# talking about the same thing.
It could be like 
- Decorate all parameters/returns with requires/ensures != null with [NotNull]
- Decorate all parameters/returns without requires/ensures != null with [CanBeNull]

With this used as a bulk operation it would be a piece of cake to convert a "CC only" library to support both CC and "R# only" users with proper information.

**Comment by SergeyTeplyakov** [comment]
@t-denis Do you mean to add something similar to current "combo" actions that adds preconditions to all nullable parameters? 

@tom-englert It could be useful to have this feature that would help to sync CC and R# attributes.

**Comment by t-denis** [comment]
Exactly.

But the idea of processing the entire file/project/solution is also very interesting.
